### PR TITLE
fix: Usage reporting format

### PR DESCRIPTION
**Pull request by benjackwhite**
## Problem

Our updates changed the organization usage report format and event name which is not ideal as this is used for reporting.

## Changes

* Reverts to the old event name and structure
* Adds a command for us to run trigger it for previous dates

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: frontend/src/scenes/billing/v2/billingLogic.ts [hunk 1]
```diff
@@ -96,9 +96,10 @@ export const billingLogic = kea<billingLogicType>([
         billingAlert: [
             (s) => [s.billing],
             (billing): BillingAlertConfig | undefined => {
-                console.log(billing?.products)
-
-                const productOverLimit = billing?.products.find((x) => {
+                if (!billing) {
+                    return
+                }
+                const productOverLimit = billing.products.find((x) => {
```
Comment (kappa90) [inline frontend/src/scenes/billing/v2/billingLogic.ts:102]: Why is this in this PR?

File: posthog/models/session_recording_event/util.py [hunk 2]
```diff
@@ -52,6 +52,18 @@ def create_session_recording_event(
     return str(uuid)
 
 
+def get_recording_count_for_team(team_id: Union[str, int]) -> int:
```
Comment (kappa90) [inline posthog/models/session_recording_event/util.py:55]: I had left this one out as it wouldn't really be a lifetime value, but only what's in storage before expiration

Comment (benjackwhite) [reply posthog/models/session_recording_event/util.py:55]: Yeah I think its fine tbh. I'll change the key to be "total" so it is not confused with actual "lifetime". But still useful to have as then we can get an idea of total usage on self-hosted for example

File: frontend/src/scenes/billing/v2/billingLogic.ts (hunk 1 above)
Comment (benjackwhite) [reply frontend/src/scenes/billing/v2/billingLogic.ts:102]: Ah this is just a quick fix for a console log. Didn't realise it was this PR but safe to go in

**Review by kappa90 — APPROVED** [review]

**Pull request merged**